rAmAnuja-sampradAyaH/tattvam/venkaTa-nAtha-shAkhA/venkaTanAthaH/nixepa-raxA/02_pUrvapaxaH/04_laxaNAnupapattisamarthanam.md

```diff
@@ -2,37 +2,43 @@
 title = "04 लक्षणानुपपत्तिसमर्थनम्"
 
 +++
-लक्षणानुपपत्तेश्च ।  
-तथा हि - अनन्यसाध्ये इत्यादिना प्रपत्तिः शरणागतिरित्यनर्थान्तरमिष्यते ।  
+ लक्षणानुपपत्तेश्च ।  
+तथा हि - 'अनन्यसाध्ये' इत्यादिना प्रपत्तिः शरणागतिरित्यनर्थान्तरमिष्यते ।  
 तथा च पठ्यते  
-निक्षेपापरपर्यायो न्यासः पञ्चाङ्गसंयुतः । संन्यासस्त्याग इत्युक्तः शरणागतिरित्यपि ॥ इति ।  
-ततश्च प्रपत्तिलक्षणं शरणागतिशब्दप्रयोगविषयसर्वोदाहरणनिष्ठमनितरनिष्ठं च प्रतिष्ठाप्यम् । तथा च दृश्यते, अनन्यसाध्ये स्वाभीष्ठे महाविश्वासपूर्वकम् । तदेकोपायतायाञ्चा प्रपत्तिः शरणागतिः ॥  
-इति सामान्यलक्षणमिति चेन्न, अव्याप्तेः, कपोतोपाख्याने तदभावात् । श्रूयते हि कपोतेन शत्रुः शरणमागतः इत्यादिभिः शरणशब्दवाच्यगृहोपलक्षितसमीपागमनमात्रमेव प्रतीयते ॥ न तत्र मुख्या शरणागतिरिति चेन्न, शरणागतसंरक्षणनिदर्शनतया शरण्यादिभिर्निर्दिश्यमानप्रसिद्धोदाहरणस्याप्यमुख्यत्वे मुख्यस्यैवासिद्धिप्रसङ्गात् । गृहागतो न परित्याज्य इति स्फुटोक्तिनिदर्शनाच्च ॥ शरणागतिस्तत्रापि कल्प्यत इति चेन्न, तत्र तथाविधलिङ्गाभावात् । समीपागतस्य परिक्लिष्टस्य परिक्लेशादेव रक्षोपायसाकाङ्क्षत्वं कल्प्यत इति चेत्, किं रक्षणेच्छामात्रं शरणागतिः? हन्त तर्हि तापत्रयाभिहताः  
-संसारिणस्तन्निवृत्तिसाकाङ्क्षाः संनिहिताश्च रक्षितुर्व्याप्तस्य सर्वेश्वरस्येति सर्वेऽपि जन्तवः शरणागता इति सर्वेषां सर्वदुःखोच्छेदप्रसङ्गः । महाविश्वासोपायत्वप्रार्थनादिनैरर्थक्यप्रसङ्गश्च । कपोतं प्रत्यश्रुताऽपि बाधकाभावात्किरातस्याप्यपेक्षापर्याययाञ्छा कल्प्यत इति चेन्न, तिर्यङ्मात्रतया प्रतिपन्ने कपोतेऽनुन्मत्तस्य तथाविधप्रतिकूलतमस्य स्वरक्षणज्ञानशक्तीच्छासंभावनाऽसंभवेन प्रार्थनारूपबुद्धिविशेषकल्पनस्य बाधितत्वात् ।
-
-एवं गोप्तृत्ववरणेऽप्यव्याप्तिः । राघवं शरणं गतः इत्यादिषु गोप्तृत्ववरणमेव ह्यौचित्यादवसीयते । 
- अभियाचाम वैदेहीमेतद्धि मम रोचते । अलमेषा परित्रातुं राक्षस्यो महतो भयात् ॥  
-इत्यादिभिश्चेदमवगम्यते । तदर्थतत्त्ववेदिभिश्चोक्तम् - मातर्मैथिलि राक्षसीस्त्वयि तथैवार्द्रापराधास्त्वया रक्षन्त्या इत्यादि । दूरगतद्रौपदीप्रपदनमपि - रक्ष मां शरणागतामित्येवंरूपमेव प्रसिद्धम् ॥  
+> निक्षेपापरपर्यायो न्यासः पञ्चाङ्गसंयुतः । संन्यासस्त्याग इत्युक्तः शरणागतिरित्यपि ॥ इति ।
```

Comment:
"इति" इति नोद्धरणाङ्गम्। तद् भिन्नायां पङ्क्तौ निवेशनीयम्। एवं च उद्धरणारम्भात् पूर्वम् एका रिक्ता पङ्क्तिर् इष्यते। इमौ दौषौ अन्यत्रापि स्याताम् - परीक्ष्य समीक्रियताम्। 

https://sanskrit.github.io/groups/dyuganga/projects/text/proofreading/editing/markdown/Quotes/#what-is-a-quote इत्यत्र प्रथमम् उदाहरणं दृश्यताम्।